packages/components/src/components/autosizer/index.js

```diff
@@ -0,0 +1,4 @@
+import AutoSizer from 'react-virtualized/dist/es/AutoSizer';
```

Comment:
Why is it imported from `dist/es`?

Reply:
Ok, seems like it's required for reducing bundle size, due to lack of / non-working tree-shaking. Can you add a comment @aaron-binary ?

Reply:
Will add a comment, in general it's better to directly import components like this so we only bundle what we need. I believe we currently face an issue in components/shared where we bundle everything if we only import a single function or component. Not sure about the tree-shaking part.